src/main/java/org/team4159/frc/robot/RobotContainer.java

```diff
@@ -1,23 +1,53 @@
 package org.team4159.frc.robot;
 
 import edu.wpi.first.wpilibj.Joystick;
+import edu.wpi.first.wpilibj.smartdashboard.SendableChooser;
+import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
+import edu.wpi.first.wpilibj2.command.Command;
+import edu.wpi.first.wpilibj2.command.InstantCommand;
 import edu.wpi.first.wpilibj2.command.RunCommand;
+import edu.wpi.first.wpilibj2.command.button.JoystickButton;
 
+import org.team4159.frc.robot.commands.auto.BlueAuto;
+import org.team4159.frc.robot.commands.auto.FollowTrajectory;
+import org.team4159.frc.robot.commands.characterization.DrivetrainCharacterization;
 import org.team4159.frc.robot.subsystems.Drivetrain;
 
 import static org.team4159.frc.robot.Constants.*;
 
 public class RobotContainer {
-  private final Drivetrain drivetrain = new Drivetrain();
+  public final Drivetrain drivetrain = new Drivetrain();
 
   private final Joystick left_joy = new Joystick(CONTROLS.LEFT_JOY);
   private final Joystick right_joy = new Joystick(CONTROLS.RIGHT_JOY);
 
+  private final Command test_auto = new FollowTrajectory(Trajectories.TEST_TRAJECTORY, drivetrain);
```

Comment:
TODO in the future, find a clean to way to put these in a different class, but ok for now